src/app/template/shared/iso.model.ts

```diff
@@ -2,15 +2,8 @@ import { BaseTemplateModel } from './base-template.model';
 import { Taggable } from '../../shared/interfaces/taggable.interface';
 
 
-export class Iso extends BaseTemplateModel implements Taggable {
-  public resourceType = 'ISO';
-  public path = 'iso';
-
-  public bootable: boolean;
-  public checksum: string;
-  public size: number;
-
-  public get isTemplate(): boolean {
-    return false;
-  }
+export interface Iso extends BaseTemplateModel, Taggable {
```

Comment:
Please remove the Taggable interface. We need to explicitly write all properties. This interface will be removed in future.